dataconfig/parsers.py

```diff
@@ -330,6 +367,25 @@ def _spec_to_type(
             )
         )
     )  # chain dict.items() and create namespace
+    '''
+    
+    ns_list = chain(
+        *glom(
+            value.values(),
+            [(
+                {
+                    "v": (Coalesce("validator", default_factory=dict)),
+                    "t": (Coalesce("transformation", default_factory=dict))
+                },
+                T.values(),
+                list
+            )],
+        )
+    )
+
+    ns = {}
+    [ns.update(f) for f in ns_list]    
+    
```

Comment:
I don't think this is correct.  I have the feeling this should crash.

Reply:
thanks for the remark! i will change accordingly.
i originally tried the last approach you describe but my glom expression gives me a list of dicts and i have no idea how to get a list of tuples instead... any ideas?

Reply:
Recall that whatever is inside a `tuple` is applied recursively; so `({...}, T.values(),  tuple}` would give you `dict` -> `dict_values` -> `tuple`, and since everything is nested inside a list, you get a list of tuples.